mapfile.csv

```diff
@@ -207,6 +207,10 @@ GenuineIntel-6-B6,V0,/GRR/metrics/grandridge_metrics.json,metrics,,,
 GenuineIntel-6-BD,V1.06,/LNL/events/lunarlake_skymont_core.json,hybridcore,0x20,0x000003,Atom
 GenuineIntel-6-BD,V1.06,/LNL/events/lunarlake_lioncove_core.json,hybridcore,0x40,0x000003,Core
 GenuineIntel-6-BD,V1.06,/LNL/events/lunarlake_uncore.json,uncore,,,
+GenuineIntel-6-C5,V1.03,/ARL/events/arrowlake_skymont_core.json,hybridcore,0x20,0x000003,Atom
+GenuineIntel-6-C5,V1.03,/ARL/events/arrowlake_crestmont_core.json,hybridcore,0x20,0x000002,Atom
```

Comment:
The converter script uses the 'Atom' value in the last column to select the PMU to program events on for hybrid systems. I think the crestmont one needs to be something like LowPower to differentiate from the skymont atom PMU:
https://lore.kernel.org/lkml/0525e13d-7439-4b42-af69-85aa33f5d3c8@linux.intel.com/
https://github.com/intel/perfmon/blob/main/scripts/create_perf_json.py#L1891

Reply:
Thanks Ian. I updated the Crestmont entry to use `LowPower_Atom`.